### PR TITLE
Add command to generate a system check

### PR DESCRIPTION
**Pull request by tsteur**
### Description:

Needed for https://github.com/matomo-org/developer-documentation/pull/498

### Review

* [ ] Functional review done
* [ ] Potential edge cases thought about (behavior of the code with strange input, with strange internal state or possible interactions with other Matomo subsystems)
* [ ] Usability review done (is anything maybe unclear or think about anything that would cause people to reach out to support)
* [ ] Security review done [see checklist](https://developer.matomo.org/guides/security-in-piwik#checklist)
* [ ] Code review done
* [ ] Tests were added if useful/possible
* [ ] Reviewed for breaking changes
* [ ] Developer changelog updated if needed
* [ ] Documentation added if needed
* [ ] Existing documentation updated if needed

### CODE REVIEW

**Comment by tsteur** [comment]
Because the example plugin is neither in the release nor do we actually add the diagnostic check this should be fine to merge in 4.4.1 maybe. Otherwise can do in Matomo 4.5.0. Added some documentation in https://github.com/matomo-org/developer-documentation/pull/498

File: plugins/CoreConsole/Commands/GenerateSystemCheck.php [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+                                'ExamplePlugin'  => $pluginName,
+        );
+
+        $whitelistFiles = array('/Diagnostic', '/Diagnostic/ExampleCheck.php');
+
+        $this->copyTemplateToPlugin($exampleFolder, $pluginName, $replace, $whitelistFiles);
+
+        $this->writeSuccessMessage($output, array(
+            sprintf('plugins/%s/Diagnostic/%s.php for %s generated.', $pluginName, $className, $pluginName),
+            'You should now implement the method called <comment>execute</comment> in API.php',
```
Comment (sgiehl) [inline plugins/CoreConsole/Commands/GenerateSystemCheck.php]:
```suggestion
            sprintf('You should now implement the method called <comment>execute</comment> in %.php', $className),
```

**Review by sgiehl — CHANGES_REQUESTED** [review]

File: plugins/CoreConsole/Commands/GenerateSystemCheck.php (hunk 1 above)
Comment (diosmosis) [reply plugins/CoreConsole/Commands/GenerateSystemCheck.php:51]: Should that be `%s`?

**Comment by tsteur** [comment]
Thanks @sgiehl  👍  applied the patch

**Pull request merged**